uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/LeaveRequestTest.php

```diff
@@ -4366,6 +4367,61 @@ public function testTheTimeForFromAndToDateOfLeaveRequestIsAddedCorrectlyWhenLea
     $this->assertEquals($toDate->format('Y-m-d') ." 23:59:00", $leaveRequest->to_date);
   }
 
+  public function testTheTimeForFromAndToDateOfLeaveRequestStaysUntouchedWhenRequestIsTOIL() {
+    $absenceType = AbsenceTypeFabricator::fabricate([
+      'allow_accruals_request'   => true,
+      'max_leave_accrual'        => 10,
+      // The calculation unit is in days since otherwise a check for
+      // calculation unit in days will take action and this test will be invalid
+      'calculation_unit'         => 1,
+      'allow_accrue_in_the_past' => true
+    ]);
+    $expectedFromDate = '2016-01-08 01:23:45';
+    $expectedToDate = '2016-01-10 12:34:56';
+    $fromDate = new DateTime($expectedFromDate);
+    $toDate = new DateTime($expectedToDate);
+    $contactID = 1;
+    $this->registerCurrentLoggedInContactInSession($contactID);
```

Comment:
Don't think we need this, test should still work without it since we are not checking permissions.

Reply:
Done.

Reply:
![image](https://user-images.githubusercontent.com/3973243/37533487-e131b540-2939-11e8-98b5-4ef071c0e3aa.png)